src/POGOProtos/Networking/Requests/RequestType.proto

```diff
@@ -7,7 +7,7 @@ package POGOProtos.Networking.Requests;
 enum RequestType {
 	METHOD_UNSET = 0;                    // No implementation required
 	GET_PLAYER = 2;                      // Implemented [R & M]
-	GET_INVENTORY = 4;                   // Implemented [R & M]
+	GET_HOLO_INVENTORY = 4;              // Implemented [R & M]
```

Comment:
Why?

Reply:
I don't know about the why, but that's what the real app is doing so I choose to keep sync with the real app.

Reply:
Alright, that's good enough for me 👍